meta/autoid/errors.go

```diff
@@ -47,6 +47,8 @@ const (
 	AutoRandomNonPositive = "the value of auto_random should be positive"
 	// AutoRandomAvailableAllocTimesNote is reported when a table containing auto_random is created.
 	AutoRandomAvailableAllocTimesNote = "Available implicit allocation times: %d"
+	// AutoRandomExplicitInsertDisabledErrMsg is reported when auto_random column value is explicitly specified, but the session var 'allow_auto_random_explicit_insert' is false.
+	AutoRandomExplicitInsertDisabledErrMsg = "Explicit insertion on auto_random column is disabled by default. Try to set @@allow_auto_random_explicit_insert = true."
```

Comment:
```suggestion
	AutoRandomExplicitInsertDisabledErrMsg = "Explicit insertion on auto_random column is disabled. Try to set @@allow_auto_random_explicit_insert = true."
```
He might have set it to `false` manually.